opennms-services/src/main/java/org/opennms/netmgt/vacuumd/AutomationProcessor.java

```diff
@@ -240,7 +240,7 @@ public Action getAction() {
         }
 
         String getActionSQL() {
-            return getAction().getStatement().getContent();
+            return Optional.ofNullable(getAction()).map(s -> s.getStatement()).map(c -> c.getContent()).orElse("");
```

Comment:
What's the expected behavior when this returns an empty string vs a null?

Reply:
The difference is empty String does not cause any error. I was thinking return  String "Unknown" may make it more readable.

Reply:
Looks like it still tries to execute the statement, which is empty - I'd consider changing the logic to avoid the statement execution altogether instead.

Reply:
Sure, I can do that.